ruby_connectors/spec/temporal/workflows/read_api_data_workflow_spec.rb

```
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe RubyConnectors::Temporal::Workflows::ReadApiDataWorkflow do
```

rubocop (1.68.0)

[convention] Spec path should end with `ruby_connectors/temporal/workflows/read_api_data_workflow*_spec.rb`.

(RSpec/SpecFilePathFormat)